corehq/apps/users/views/mobile/users.py

```diff
@@ -844,7 +855,7 @@ def _build_commcare_user(self):
 
         return CommCareUser.create(
             self.domain,
-            format_username(username, self.domain),
+            username,
```

Comment:
For sure.  `format_username` isn't gone - I added `clean_mobile_worker_username` to the `clean_username` method, and that function includes the functionality of `format_username`.  Actually when I first added that, I was getting users like `ethan@test.commcarehq.org@test.commcarehq`, until I realized I needed to drop this call.  I added that clean function because we were only validating username uniqueness on the client-side.